packages/currency/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@helium/currency",
-  "version": "4.9.0",
+  "version": "5.0.0",
```

Comment:
You dont need to change this manually, When we run a release it will auto update. It should be reverted